### DISCUSSION TITLE
Pull Requests Not Merging

### DISCUSSION THREAD

**Discussion by gburkecw** [community/community#77115] [Pull Requests]
### Select Topic Area

Bug

### Body

This issue just started happening to us in the last hour, we auto-merged pull requests earlier today. We currently cannot merge pull requests at all:

![image](https://github.com/community/community/assets/113136756/bda08932-266c-46cd-b713-56e63951d927)

**Comment by co60ca** [comment]
![Screenshot 2023-11-27 at 2 41 56 PM](https://github.com/community/community/assets/6857411/0ec7ff20-31d5-4e96-b7c8-a3935eb7a68e)
Possibly related? Maybe the last commit hasn't been processed? Do you have this as well?

**Reply by beerlington** [reply]
They just acknowledged the issue https://www.githubstatus.com/incidents/66vhjmd266r9

**Reply by co60ca** [reply]
Thanks for relaying this. I checked githubstatus before coming here, so we had a real life race condition.

**Comment by lcn2** [comment]
Possibly related:

After successfully merging one pull request earlier this morning, our pull requests are stuck behind a "Checking merge status..." hang:

![image](https://github.com/community/community/assets/10429707/665bddef-ca52-40bc-ab3b-526f2fb0cefb)

that after several minutes turns into a "Merge status unknown":

![image](https://github.com/community/community/assets/10429707/3d95733f-9ebf-4b1d-b447-d1be1354383f)

Clicking the ((Recheck merge status)) sends us right back to the "Checking merge status..." hang.

**Comment by kodebuilder-com** [comment]
Facing same issue from last half n hour

![Screenshot 2023-11-28 015041](https://github.com/community/community/assets/5411251/f0eadc7b-c0d1-442e-a795-8295186f60a5)
